api/models/post.model.js

```diff
@@ -20,5 +21,4 @@ const database = {
         }
     ]
 }
-
 module.exports = database 
```

Comment:
Going forward, you should use Eslint to format your codebase and remove this end of line problems

Reply:
I already npm install eslint and did the setup but its not working

Reply:
Maybe I got the setup wrong

Reply:
Try running this command, it will install all dependencies and setup a configuration file for eslint.
`npx install-peerdeps --dev eslint-config-airbnb`